### PR TITLE
Update cl.m3u

### PR DESCRIPTION
**Pull request by adamford21**

### CODE REVIEW

File: channels/cl.m3u [hunk 1]
```diff
@@ -152,4 +152,10 @@ http://unlimited6-cl.dps.live/vtv/vtv.smil/playlist.m3u8
...
 http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
 #EXTINF:-1 tvg-id="ZappingMusicHD.cl" tvg-name="Zapping Music HD" tvg-country="CL" tvg-language="" tvg-logo="https://i.imgur.com/lHvoKdw.jpg" group-title="Music",Zapping Music HD
-https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8
+https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8#EXTINF:-1 tvg-id="TVOSanVicente.cl" tvg-name="TVO San Vicente" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/fvnO1X9.png" group-title="",TVO San Vicente
+https://unlimited2-cl-movistar.dps.live/tvo/tvo.smil/tvo/livestream1/chunks.m3u8
+#EXTINF:-1 tvg-id="Canal9Concepción.cl" tvg-name="Canal 9" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/UnWMRrT.png" group-title="",Canal 9
+https://unlimited2-cl.dps.live/c9/c9.smil/c9/livestream2/chunks.m3u8
+#EXTINF:-1 tvg-id="SantiagoTV.cl" tvg-name="Santiago TV" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9K9lQl6.jpg" group-title="",Santiago TV
+https://stv.janus.cl/playlist/stream.m3u8
+
```
Comment (Dum4G) [inline channels/cl.m3u:161]: Please remove that empty line so we could pass the linter check

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by adamford21 — COMMENTED** [review]
line removed

File: channels/cl.m3u [hunk 2]
```diff
@@ -152,4 +152,9 @@ http://unlimited6-cl.dps.live/vtv/vtv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="VTVVinaValparaisoHD.cl" tvg-name="VTV Vina - Valparaiso HD" tvg-country="CL" tvg-language="" tvg-logo="http://images.eltelon.com/cache/media/4151/square_150x150-2.jpg" group-title="",VTV Vina - Valparaiso HD
 http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
 #EXTINF:-1 tvg-id="ZappingMusicHD.cl" tvg-name="Zapping Music HD" tvg-country="CL" tvg-language="" tvg-logo="https://i.imgur.com/lHvoKdw.jpg" group-title="Music",Zapping Music HD
-https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8
+https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8#EXTINF:-1 tvg-id="TVOSanVicente.cl" tvg-name="TVO San Vicente" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/fvnO1X9.png" group-title="",TVO San Vicente
```
Comment (Dum4G) [inline channels/cl.m3u:155]: Please make a new line after playlist.m3u8. In other words separate the link and extinf

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/cl.m3u [hunk 3]
```diff
@@ -153,3 +153,9 @@ http://unlimited6-cl.dps.live/vtv/vtv.smil/playlist.m3u8
 http://cdn.streamingmedia.cl:1935/live/vtvvina/playlist.m3u8
 #EXTINF:-1 tvg-id="ZappingMusicHD.cl" tvg-name="Zapping Music HD" tvg-country="CL" tvg-language="" tvg-logo="https://i.imgur.com/lHvoKdw.jpg" group-title="Music",Zapping Music HD
 https://zmlive.zappingtv.com/zm_free/zm.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="TVOSanVicente.cl" tvg-name="TVO San Vicente" tvg-country="CL" tvg-language="Spanish" tvg-logo="https://i.imgur.com/fvnO1X9.png" group-title="",TVO San Vicente
```
Comment (adamford21) [inline channels/cl.m3u:156]: Separated the link from the #EXTINF

**Comment by Dum4G** [comment]
Thanks for your contributions!

**Review by Dum4G — APPROVED** [review]

**Pull request merged**